src/CVXOPT.jl

```diff
@@ -299,28 +203,21 @@ function julia_to_cvxopt(A)
   if issparse(A)
     J = zeros(Int64, length(A.rowval));
     for i = 1:size(A,2)
-      J[A.colptr[i]:A.colptr[i+1]-1] .= i;
+      J[A.colptr[i]:A.colptr[i+1]-1] .= i - 1;
     end
-    Ap = cvxopt[:spmatrix](PyVector(A.nzval),PyVector(A.rowval.-1),PyVector(J.-1),(size(A,1),size(A,2)));
+    I = A.rowval .- 1
+    V = A.nzval
+    Ap = @pycall  cvxopt.spmatrix(py"list($$V)"o,py"list($$I)"o,py"list($$J)"o,(size(A,1),size(A,2)))::PyObject;
   elseif isempty(A)
     Ap = pybuiltin("None");
   else
     sA = size(A)
     if length(sA) == 1
       sA = (sA[1],1)
     end
-    Ap = cvxopt[:matrix](A[:],sA);
+    Ap = @pycall cvxopt.matrix(py"list($(A[:]))"o,sA)::PyObject;
```

Comment:
You can use `PyCall.array2py(vec(A))` to convert to a flattened Python list without making multiple copies.   I should  probably export a `pylist` function to do this.

Reply:
Thanks for pointing this out—I was looking for something like that, but I didn't find it in the PyCall README.